src/Request.js

```diff
@@ -187,6 +187,20 @@ export function jsonp (url, params, callback, context) {
   script.type = 'text/javascript';
   script.src = url + '?' + serialize(params);
   script.id = callbackId;
+  script.onerror = function (error) {
+    if (error && window._EsriLeafletCallbacks[callbackId] !== true) {
+      // Can't get true error code: it can be 404, or 401, or 500
+      var err = {
+        error: {
+          code: 500,
```

Comment:
if we don't know what the actual error was, best not to assign a code arbitrarily.

what about something like this?
```js
var err = {
  message: 'An unknown error occurred'
}
```

Reply:
Ok, it makes sense. I've just tried to save error contract as
```javascript
var err = {
  error: {
     code: 'need something undefined',
     message: 'An unknown error occurred'
  }
}
```

May be close this PR and make another with correct username and message fix?
And what about failed tests?

Reply:
> I've just tried to save error contract 

meh, [this guy](https://stackoverflow.com/questions/5334413/parsing-jsonp-response-in-javascript-when-4xx-or-5xx-http-error-code-is-present) seems to agree with you, so i can live with a 500 if you update the message. 

> May be close this PR and make another with correct username

no need to close this PR. just push another commit to the same branch (and rebase if it'd make you happy).

> what about failed tests?

those weren't your fault. i restarted the build and am seeing 💚 now.